app/services/bgs/payment_service.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
```

Comment:
@sethdarragile6 Please resolve merge conflicts

Reply:
@RachalCassity oh wow, looks like someone swooped in and totally refactored the logging- making this PR totally irrelevant! 😆😢 C'est la vie. Closing out!